### ISSUE TITLE
I'm getting vscode.merge-conflict errors in the extension host log

### ISSUE THREAD

**Issue by sklop** [cdr/code-server#2740]
- Web Browser: Chrome
- Local OS: MacOS
- Remote OS: AWS linux 2
- Remote Architecture: x86_64
- `code-server --version`: 3.9.0 fc6d123da59a4e5a675ac8e080f66e032ba01a1b

I'm receiving vscode.merge-conflict errors in the extension host log repeatedly and it seems to be preventing certain extensions from working

```
[2021-02-18 00:53:29.704] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode.js-debug {"startup":false,"extensionId":{"value":"ms-vscode.js-debug","_lower":"ms-vscode.js-debug"},"activationEvent":"onCommand:extension.js-debug.clearAutoAttachVariables"}
[2021-02-18 00:53:29.704] [exthost] [info] ExtensionService#loadCommonJSModule file:///usr/lib/code-server/lib/vscode/extensions/ms-vscode.js-debug/src/extension.js
[2021-02-18 00:53:29.833] [exthost] [error] [vscode.merge-conflict] provider FAILED
[2021-02-18 00:53:29.833] [exthost] [error] Error: Unable to retrieve document from URI 'file:///home/sklop/code-server/User/settings.json'
	at t.ExtHostDocuments.getDocument (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:735:184)
	at b.provideCodeLenses (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:752:624)
	at /usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:774:370
	at q._withAdapter (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:773:10)
	at q.$provideCodeLenses (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:774:348)
	at m._doInvokeHandler (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:893:926)
	at m._invokeHandler (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:893:618)
	at m._receiveRequest (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:892:201)
	at m._receiveOneMessage (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:891:6)
	at /usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:889:136
	at l.fire (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:801)
	at v.fire (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:234:448)
	at /usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1095:141
	at l.fire (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:801)
	at v.fire (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:234:448)
	at t.PersistentProtocol._receiveMessage (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:239:663)
	at /usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:236:563
	at l.fire (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:801)
	at p.acceptChunk (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:231:897)
	at /usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:231:249
	at l.fire (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:801)
	at v._acceptChunk (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:244:563)
	at /usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:242:671
	at Socket.t (/usr/lib/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:241:463)
	at Socket.emit (events.js:315:20)
	at addChunk (_stream_readable.js:295:12)
	at readableAddChunk (_stream_readable.js:271:9)
	at Socket.Readable.push (_stream_readable.js:212:10)
	at TCP.onStreamRead (internal/stream_base_commons.js:186:23)
```

**Comment by jsjoeio** [comment]
Thanks for opening this issue! @sklop 

Do you mind providing steps to reproduce this?

> it seems to be preventing certain extensions from working

Can you provide a list of extensions that you're referring to?

**Comment by sklop** [comment]
Hi @jsjoeio,
The error comes up when code-server is initially hit by the browser.  Your hint on the plugins helped me figure out what was going on.  I had installed a few new plugins recently.  So I went through and starting removing them one by one until I found the culprit.  The plugin causing the issue iis HTMLHint.  
![image](https://user-images.githubusercontent.com/6834515/109227968-03e33400-7776-11eb-9c3d-6e4e1aece551.png)

When the error happened it seemed to cause other plugins to have the same error.  But once I removed it, the rest of the plugins loaded normally

Here is my list of plugins:
![image](https://user-images.githubusercontent.com/6834515/109228487-bddaa000-7776-11eb-8d22-1783f04ef3e6.png)

![image](https://user-images.githubusercontent.com/6834515/109228535-d1860680-7776-11eb-9ec5-19dae3dfc5eb.png)

![image](https://user-images.githubusercontent.com/6834515/109228570-dd71c880-7776-11eb-99d4-03745249f655.png)


Thanks!
Shaun
